Cargo.toml

```diff
@@ -16,11 +16,11 @@ members = [
     "lightning-macros",
     "lightning-dns-resolver",
     "lightning-liquidity",
+    "lightning-transaction-sync",
```

Comment:
Given BDK is about to bump their MSRV to 1.85, let's not bother with this if we need to move it out again shortly.